sway/sway.5.scd

```diff
@@ -546,6 +546,8 @@ The default colors are:
 
 *for_window* <criteria> <command>
 	Whenever a window that matches _criteria_ appears, run list of commands.
+	A list of _criteria_ for currently opened windows may be obtained via 
+	swaymsg -t get_tree.
```

Comment:
Hmm, not really: `swaymsg -t get_tree` doesn't return criteria (ie. conditions), it returns window properties.

This addition should probably be moved to the *CRITERIA* section (since criteria isn't only used in `for_window`), and say that attributes for currently opened windows may be obtained via `swaymsg -t get_tree`.

Reply:
Okay edited